app/controllers/giveaways_controller.rb

```diff
@@ -55,10 +55,6 @@ def edit
     def show
         @giveaway = Giveaway.find(params[:id])
         if !@giveaway.isActive
```

Comment:
Should `isActive` be changed to `active?` in the db or is a getter method `active?` ok?